app/code/Magento/Catalog/Test/Mftf/Test/StoreFrontSimpleProductWithSpecialAndTierDiscountPriceTest.xml

```diff
@@ -26,22 +26,27 @@
             <createData entity="tierProductPriceDiscount" stepKey="addTierPrice">
                 <requiredEntity createDataKey="createProduct"/>
             </createData>
-
-            <createData entity="specialProductPrice2" stepKey="addSpecialToSimpleProduct">
-                <requiredEntity createDataKey="createProduct"/>
-                <field key="price">65.00</field>
-            </createData>
-            <magentoCLI command="cron:run --group=index" stepKey="runCronIndex"/>
         </before>
         <after>
             <deleteData createDataKey="createProduct" stepKey="deleteSimpleProduct"/>
             <deleteData createDataKey="createCategory" stepKey="deleteCategory"/>
         </after>
 
+        <actionGroup ref="LoginAsAdmin" stepKey="LoginAsAdmin"/>
+
+        <actionGroup ref="GoToProductPageViaIDActionGroup" stepKey="openAdminProductEditPage">
+            <argument name="productId" value="$createProduct.id$"/>
+        </actionGroup>
+
+        <actionGroup ref="AddSpecialPriceToProductActionGroup" stepKey="addSpecialPriceToProduct">
```

Comment:
don't you need to define on which product you want to add special price?

Reply:
No, action group expects that I've already on the product edit page. Please see
https://github.com/magento/magento2/blob/a25c107b7ec90fb191317810097ae1fabbb91879/app/code/Magento/Catalog/Test/Mftf/ActionGroup/AddSpecialPriceToProductActionGroup.xml#L12-L17

Reply:
Got it. Thank you!